app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'application', 'https://fonts.googleapis.com/css?family=Chonburi|Merriweather|Playfair+Display+SC', type='text/css' %>
```

Comment:
not quite. Remove "application" (i.e. 'application.scss').
see http://apidock.com/rails/ActionView/Helpers/AssetTagHelper/stylesheet_link_tag